tests/misc_test.go

```diff
@@ -623,3 +623,15 @@ func TestTypeConversion(t *testing.T) {
 		t.Fail()
 	}
 }
+
+func TestSliceOutOfRange(t *testing.T) {
```

Comment:
What do you think about moving this test and placing it next to `TestSliceOfString`, since it's quite similar and related?

Reply:
Done.